script/deploy/2_ConfigureOFT.s.sol

```diff
@@ -34,6 +34,7 @@ contract ConfigureOFT is BaseScript {
             configureEnforcedOptions(dstChainIds);
             configureDVNs(dstChainIds);
             configureExecutor(dstChainIds);
+            configureDelegate();
```

Comment:
This step was missing from the configuration.
This sets the delegate for the oft adapter on the lz endpoint. 

In most cases, the lz endpoint just checks if the sender if the oftAdapter.owner(), but in some config changes it checks if the sender is the delegate.

Reply:
do we need to do this for current deployments?

What is the implication if we don't have this?

Reply:
When we need to update the config, particular calling `setConfig` on the lz enpoint, it can only be called by the delegate.

But since we are doing it via the multisig batch tx, it will be added as the first tx (see line 310 in VerifyOFT script) and therefore, all subsequent config changes will work fine.